test/framework/easyconfig.py

```diff
@@ -1559,7 +1560,6 @@ def test_quote_str(self):
             'foo': '"foo"',
             'foo\'bar': '"foo\'bar"',
             'foo\'bar"baz': '"""foo\'bar"baz"""',
-            "foo'bar\"baz": '"""foo\'bar"baz"""',
```

Comment:
@Flamefire Why is this test case removed?

Reply:
Because it is literally the same as the one above

Reply:
(I think my linter complained)

Reply:
Right, it's silly indeed...